modules/darwin/src/main/ts/ephox/darwin/queries/CellOpSelection.ts

```diff
@@ -0,0 +1,40 @@
+import { Arr, Fun, Optional } from '@ephox/katamari';
+import { Attribute, SugarElement } from '@ephox/sugar';
+import { Structs, RunOperation } from '@ephox/snooker';
+import { Selections } from '../selection/Selections';
+import * as TableSelection from '../api/TableSelection';
+import { Ephemera } from '../api/Ephemera';
+import * as SelectionTypes from '../selection/SelectionTypes';
+
+// Return an array of the selected elements
+const selection = (selections: Selections): SugarElement[] =>
+  SelectionTypes.cata(selections.get(),
+    Fun.constant([]),
+    Fun.identity,
+    Arr.pure
+  );
+
+const unmergable = (selections: Selections): Optional<SugarElement[]> => {
+  const hasSpan = (elem: SugarElement<Element>) => (Attribute.has(elem, 'rowspan') && parseInt(Attribute.get(elem, 'rowspan') as string, 10) > 1) ||
```

Comment:
This would possibly be better to do something like this maybe?
```
const hasSpan = (elem: SugarElement<Element>, type: 'colspan' | 'rowspan') => Attribute.getOpt(elem, type).exists((span) => parseInt(span, 10) > 1);
const hasRowOrColSpan = (elem: SugarElement<Element>) => hasSpan(elem, 'rowspan') || hasSpan(elem, 'colspan');
```

Reply:
That'll avoid the cast and saves having to do the `has` and `get` calls.